packages.lisp

```diff
@@ -31,6 +31,13 @@
 	   #:write-conllu-to-stream   
 	   #:levenshtein
 	   #:diff
+
+	   #:diff-parent-upostag
```

Comment:
do we need to export auxiliary functions?

Reply:
The functions implemented are specific ways to perfor a diff in the sentences and building and saving the resulting confusion table. The library already exports a function which aims to make a diff of sentences (cl-conllu:diff). In my interpretation, exporting the functions adds to this type of 'diff' features. But I understand that in the moment they solve very specific needs of the team and may not add to the library as a generic tool for manipulating conllu files.